dataset/dictionary/characters.ts

```diff
@@ -2485,8 +2652,9 @@ export default [
     zhCN: "（对珊瑚宫心海或雷电将军的尊称）",
     zhTW: "（對珊瑚宮心海或雷電將軍的尊稱）",
     tags: [ "inazuma", "how-to-call" ],
-    notes: "単に Her Excellency のみで主君を指したり、Her Excellency, Sangonomiya などのように名前の前に付けて用いたりする。一般用語でも、高貴な人物に対して [Her/His/Your] Excellency が用いられる。",
-    notesZh: "单纯使用 Her Excellency 即指代君主，如 Her Excellency, Sangonomiya 这样加在名字前面。在一般用语中，对高贵的任务也使用 [Her/His/Your] Excellency 的说法。",
+    notes: "単に Her Excellency のみで主君を指したり、Her Excellency, Sangonomiya などのように名前の前に付けて用いたりする。一般用語としては「閣下」と訳されることが多く、高貴な人物に対して [Her/His/Your] Excellency が用いられる。",
```

Comment:
```suggestion
    notes: "単に Her Excellency のみを用いたり、Her Excellency, Sangonomiya のように名前の前に付けて用いられることがある。一般用語としては「閣下」と訳されることが多く、高貴な人物に対して [Her/His/Your] Excellency が用いられる。",
```

Reply:
Repeating "用いる" twice sounds a bit redundant to me.
How about the following rewrite?

```suggestion
    notes: "自らの主君を指す言葉であり、単に \"Her Excellency\" のみ、または \"Her Excellency, Sangonomiya\" のように名前の前に付けて用いる。一般用語としては「閣下」と訳されることが多く、高貴な人物に対して [Her/His/Your] Excellency が用いられる。",
```